sabnzbd/utils/feedparser.py

```diff
@@ -529,6 +529,7 @@ class _FeedParserMixin:
         'http://www.w3.org/XML/1998/namespace':                  'xml',
         'http://www.newznab.com/DTD/2010/feeds/attributes/':     'nZEDb',
         'http://www.newznab.com/DTD/2010/feeds/attributes/':     'newznab',
+        'http://www.newznab.com/DTD/2010/feeds/attributes/':     'nntmux',
```

Comment:
Patching feedparser.py is OK, but I think a number of package creators (like JCFP) are using the already installed version. You would need to change the name to cover this (and warn package makers).